packages/react-components/src/button/src/Button.css

```diff
@@ -18,12 +18,14 @@
     cursor: pointer;
     outline: none;
     font-weight: 500;
+    justify-content: center;
```

Comment:
Doesn't make sense to me to set `justify-content` when the element display is not `flex` or `grid` ?

Reply:
This was meant to be as generic as possible, I agree that I'd rather put it on `.o-ui-button:not(.o-ui-icon-button)` which is where we set buttons to a display of `inline-grid`